### PR TITLE
Adds webpack bundle analyzer to encourage bundle minimization

### PR DESCRIPTION
**Pull request by iendjinn**
When running in development mode webpack bundle analyzer will auto start to show you how badly optimised your code is.

### CODE REVIEW

File: webpack/webpack.dev.babel.js [hunk 1]
```diff
@@ -26,6 +27,30 @@ module.exports = (function() {
     optimization: {
       splitChunks: {
         chunks: "all",
+        minSize: 0,
+        minChunks: 1,
+        maxAsyncRequests: 5,
+        maxInitialRequests: Infinity,
```
Comment (aloisklink) [inline webpack/webpack.dev.babel.js]: Why are we adjusting the parameters of `minSize`: `0`, `maxAsyncRequests` and `maxInitialRequests` from the defaults?

I feel like `minSize: 0` might start creating chunks that are too small, so the HTTP overhead is bigger than the actual chunk.

And `maxAsyncRequests: 30` and `maxInitialRequests: 30` seem like okay defaults, so we should have some sort of comment saying why we are changing them.

https://webpack.js.org/plugins/split-chunks-plugin/

**Review by aloisklink — COMMENTED** [review]
Looks good!

I do have a minor comment on why are updating defaults of the split-chunks-plugin (might be worth adding a comment), but to be honest, if this is only for the dev environment, it probably isn't super important.

**Review by aloisklink — APPROVED** [review]
Whoop, accidentally hit comment instead of approve in the earlier box. Maybe I need my coffee :coffee:

File: webpack/webpack.dev.babel.js (hunk 1 above)
Comment (iendjinn) [reply webpack/webpack.dev.babel.js]: Good catch, I've been experimenting and forgot to change them to sensible values :)

**Pull request merged**